content/blog/new-kubernetes-superpowers/index.md

```diff
@@ -0,0 +1,162 @@
+As the Kubernetes ecosystem grows, projects across the Cloud Native landscape build powerful new extensions on top of the core Kubernetes resource model via [CustomResources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/). These Custom Resources extend the Kubernetes API to provide first-class support for everything from Service Meshes to Storage extensions to Observability infrastructure and much more (including the Pulumi Kubernetes Operator!). While Pulumi offers a strongly-typed API for 100% of the core Kubernetes API, so far it has only exposed these extensions via the weakly-typed and general-purpose `CustomResource` API in Kubernetes.
+
+Today’s launch of the [`crd2pulumi`](https://github.com/pulumi/pulumi-kubernetes/tree/master/provider/cmd/crd2pulumi) tool brings the same rich support for strongly-typed APIs in TypeScript and Go (Python and .NET coming soon!) to all of these extensions. Users can generate a strongly-typed programming language API directly from the Custom Resource Definition’s OpenAPI spec, and regenerate it if and when that spec is updated. Any code using the CRD benefits from IDE tooling, strong typing, and the ease of creating new custom abstractions. These benefits become more pronounced as the complexity of the infrastructure managed by the custom resource increases. Many CRDs encourage design patterns involving fine-grained and frequently-updated resource definitions, and Pulumi’s modern infrastructure as code approach greatly simplifies managing an ever-growing ecosystem of new APIs.
+
+For example, the commonly used [cert-manager](https://cert-manager.io/docs/installation/kubernetes/) CRD extends Kubernetes with the ability to easily configure Certificate Authorities and request certificates. After generating API wrappers with `crd2pulumi`, users can interact with the Cert Manager API from code:
+
+<!-- TODO: Replace with an IDE screenshot to highlight IDE tooling and/or lang chooser. -->
+
+```ts
+import * as certificates from "./certificates"
+
+new certificates.v1beta1.Certificate("example-cert", {
+    metadata: {
+        name: "example-com",
+    },
+    spec: {
+        secretName: "example-com-tls",
+        duration: "2160h",
+        renewBefore: "360h",
+        commonName: "example.com",
+        dnsNames: [
+            "example.com",
+            "www.example.com",
+        ],
+        issuerRef: {
+            name: "ca-issuer",
+            kind: "Issuer",
+        }
+    }
+});
+```
+
+## OPA support in Pulumi CrossGuard Policy-as-Code
+
+Pulumi’s [CrossGuard](https://www.pulumi.com/docs/guides/crossguard/) policy-as-code framework enforces policy across all your modern cloud infrastructure and/or in Kubernetes - using the full expressiveness of rich programming languages. It is deeply integrated into the deployment process to ensure that policy violations are identified *before* infrastructure is deployed instead of minutes or hours after changes are made.
```

Comment:
```suggestion
Pulumi’s [CrossGuard]({{< relref “/docs/guides/crossguard” >}}) policy-as-code framework enforces policy across all your modern cloud infrastructure and/or in Kubernetes - using the full expressiveness of rich programming languages. It is deeply integrated into the deployment process to ensure that policy violations are identified *before* infrastructure is deployed instead of minutes or hours after changes are made.
```